OneSchedule/SetWebHookService.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Options;
+using OneSchedule.Settings;
+using System.Threading;
+using System.Threading.Tasks;
+using Telegram.Bot;
+
+namespace OneSchedule
+{
+    public class SetWebHookService : IHostedService
+    {
+        private readonly IOptions<WebHookSettings> _webHookOptions;
+        private readonly ITelegramBotClient _bot;
+
+        public SetWebHookService(IOptions<WebHookSettings> webHookOptions, ITelegramBotClient bot)
+        {
+            _webHookOptions = webHookOptions;
```

Comment:
It is usually better to have a field of settings class itself instead of IOptions<T>